.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+<!-- Start by describing the changes made in this Pull Request, and the reason for such changes. -->
+
+<!-- Reference any related issues or PRs here. Each issue needs the "fixes" keyword if the PR fixes more than one thing. -->
+Fixes #, See #
+
+<!-- Don't forget to update the title with something descriptive. -->
+
+### Screenshots
+
+<!-- If your change has a visual component, add a screenshot here. A "before" screenshot would also be helpful. -->
+
+### How to test the changes in this Pull Request:
+
+1.
+2.
+3.
+
+<!-- If you can, add the appropriate [Component] and [Status] labels. -->
```

Comment:
`MAYBE` - I think committers will already know to do this, and everyone else won't be able to add labels, so it'd probably be best to remove this.

Reply:
It's a good reminder, I'd rather leave it.

Reply:
Yeah, I think these templates should be used by committers also, so I agree it's a good reminder.